### PR TITLE
added a dockerfile

### PR DESCRIPTION
**Pull request by makosblade**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "test": "echo \"Error: test task not implemented\" && exit 1",
     "webpack:dev": "NODE_ENV=development webpack --config config/webpack.config.babel.js",
     "webpack:prod": "NODE_ENV=production webpack --config config/webpack.config.babel.js",
-    "webpack:server": "NODE_ENV=development webpack-dev-server --config config/webpack.config.babel.js"
+    "webpack:server": "NODE_ENV=development webpack-dev-server --config config/webpack.config.babel.js",
+    "webpack:prodServer": "NODE_ENV=production webpack-dev-server --config config/webpack.config.babel.js --host 0.0.0.0"
```
Comment (UnsungHero97) [inline package.json]: for convention's sake, can you make it `webpack:prod:server`. also, how is this a prod server if host is `0.0.0.0`?

Comment (makosblade) [inline package.json]: also, will update the string

Comment (UnsungHero97) [inline package.json]: wait... so does this mean the production server is basically the same thing as running `npm run webpack:prod:server`?

Comment (makosblade) [inline package.json]: It ends up being nginx proxied to webpack:prod:server. We can use nginx instead, this was just the fastest way to get it up and running.

Comment (UnsungHero97) [inline package.json]: that would be preferable

**Pull request merged**